inc/composer/class-installer.php

```diff
@@ -42,6 +42,7 @@ public function getInstallPath( PackageInterface $package, $framework_type = ''
 			'10up/elasticpress',
 			'altis/aws-analytics',
 			'altis/browser-security',
+			'altis/consent',
```

Comment:
Need to add the consent API JS plugin here too, just checked and it's also of the type `wordpress-plugin` but expected to be loaded from the vendor folder.

```suggestion
			'altis/consent',
			'altis/consent-api',
```

Reply:
Yep. I knew this, but for some reason I had it in my head that, because `altis-consent` was loading `altis/consent-api`, that it didn't need to go in here to replace the `wp-consent-api`. 🤦‍♂️

Reply:
Yeah it's not that clever unfortunately, no kind of inheritance logic around that.